README.rst

```diff
@@ -1,8 +1,7 @@
 `ndcube`
 ========
 
-|Latest Version| |Build Status| |Build status| |codecov| |Research
-software impact| |Powered by NumFOCUS| |Powered by Sunpy|
+|Latest Version| |Build Status| |Build status| |codecov| |Research software impact| |Powered by NumFOCUS| |Powered by Sunpy|
```

Comment:
@Cadair @nabobalis The ```|Research software impact|``` isn't actually used below which I think is why the tag is "broken".  Where should it be pointing?  Or should we just remove it?

Reply:
I removed the URL for the badge but forgot to remove it. I'd delete it.